extensions/wikia/PortableInfobox/styles/PortableInfoboxBuilder.scss

```diff
@@ -0,0 +1,37 @@
+@import 'skins/shared/color';
+@import "PortableInfoboxVariables";
+
+.portable-infobox-builder {
+	background-color: $color-page;
+	padding: 20px 50px;
+	position: relative;
+	height: 500px;
+
+	.title {
+		margin-bottom: 50px;
+	}
+
+	.publish-btn {
+		position: absolute;
+		right: 50px;
+		top: 20px;
+	}
+
+	.infobox-preview,
+	.options-pannel {
+		float: left;
+		margin-right: 50px;
+		width: $infobox-width;
+	}
+
+	.options-pannel {
+		background-color: $color-unified-module-background;
+		box-sizing: border-box;
+		color: darken($color-unified-module-background, 30%);
+		font-size: 16px;
+		overflow: hidden;
+		padding-top: 20px;
+		min-height: 249px;
```

Comment:
please fix alphabetical order :)
why there is a `249px`? Cannot be 250px? Can we maybe move it to const and use in line 8 as `height: $infobox-builder-height / 2;` if this variables are somehow connected?